client/ayon_blender/api/lib.py

```diff
@@ -771,3 +771,13 @@ def map_colorspace_name(colorspace: str) -> str:
     }
 
     return colorspace_mapping.get(colorspace, colorspace)
+
+
+def get_scene_node_tree():
```

Comment:
Not sure what the overload of this is, but it might be good idea to cache it, with blender version too maybe (like if is above 5.0.0 boolean)?